Makefile

```diff
@@ -20,11 +20,13 @@ KERNEL_CONFIG := $(KLIB_BUILD)/.config
 KERNEL_MAKEFILE := $(KLIB_BUILD)/Makefile
 CONFIG_MD5 := $(shell md5sum $(KERNEL_CONFIG) 2>/dev/null | sed 's/\s.*//')
 KBUILD_MODPOST_WARN := 1
-PKG_DISTRO_TARGETS := i915dkmsdeb-pkg i915dkmsrpm-pkg binrpm-pkg
+DEB_PKG_DISTRO_TARGETS := i915dkmsdeb-pkg bindeb-pkg
+RPM_PKG_DISTRO_TARGETS := i915dkmsrpm-pkg
+PKG_DISTRO_TARGETS := $(DEB_PKG_DISTRO_TARGETS) $(RPM_PKG_DISTRO_TARGETS)
 
```

Comment:
look like bin rpm is got missed here, please take  care in the next pr, Not blocking releases as binrpm is not the primary usage